ivadomed/scripts/prepare_dataset_vertebral_labeling.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+##############################################################
+#
+# This script is used to create a dataset with (i) mid-sagittal image and
+# (ii) heatmap of disc labels associated with the mid-sagittal image.
+#
+# Usage: python scripts/automate_training.py -p bids_path -s T2w -a -1
```

Comment:
obsolete?

Reply:
Yes! Came from a misunderstanding about automatically generated documentation

Reply:
Removed in cbdad4d